src/lighting/lighting-controller.js

```diff
@@ -400,6 +407,29 @@ class LightingController {
   }
 
   /**
+   * Uses the Hue API to change the color of all lights within range.
+   */
+  colorLights(xColor) {
+    var _self = this;
+
+    switch(xColor){
```

Comment:
Not a requirement, but if you want to support more colors we could incorporate the usage of the _[onecolor](https://www.npmjs.com/package/onecolor)_ NPM package. This library is able to convert any of the standard CSS color names into their RGB representations.